specification/maps/data-plane/Weather/preview/1.0/weather.json

```diff
@@ -564,621 +715,1772 @@
+            "$ref": "#/parameters/ApiVersion"
           },
           {
-            "value": 15,
-            "name": "Thunderstorms",
-            "description": "Thunderstorms"
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/JsonFormat"
           },
           {
-            "value": 16,
-            "name": "MostlyCloudyWithThunderstorms",
-            "description": "Mostly Cloudy with Thunderstorms"
+            "$ref": "#/parameters/WeatherYear"
           },
           {
-            "value": 17,
-            "name": "PartlySunnyWithThunderstorms",
-            "description": "Partly Sunny with Thunderstorms"
+            "$ref": "#/parameters/RequiredWeatherBasinId"
           },
           {
-            "value": 18,
-            "name": "Rain",
-            "description": "Rain"
+            "$ref": "#/parameters/RequiredWeatherGovernmentId"
           },
           {
-            "value": 19,
-            "name": "Flurries",
-            "description": "Flurries"
+            "$ref": "#/parameters/WeatherDataUnit"
           },
           {
-            "value": 20,
-            "name": "MostlyCloudyWithFlurries",
-            "description": "Mostly Cloudy with Flurries"
+            "$ref": "#/parameters/WeatherDetails"
           },
           {
-            "value": 21,
-            "name": "PartlySunnyWithFlurries",
-            "description": "Partly Sunny with Flurries"
+            "$ref": "#/parameters/WeatherRadiiGeometry"
           },
           {
-            "value": 22,
-            "name": "Snow",
-            "description": "Snow"
+            "name": "windowGeometry",
+            "description": "When true, window geometry data (geoJSON) is included in the response",
+            "in": "query",
+            "type": "boolean",
+            "default": false
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/StormForecastResponse"
+            }
           },
-          {
-            "value": 23,
-            "name": "MostlyCloudyWithSnow",
-            "description": "Mostly Cloudy with Snow"
-          },
-          {
-            "value": 24,
-            "name": "Ice",
-            "description": "Ice"
-          },
-          {
-            "value": 25,
-            "name": "Sleet",
-            "description": "Sleet"
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
+    "/weather/tropical/storms/locations/{format}": {
+      "get": {
+        "description": "**Get Tropical Storm Locations**\n\n**Applies to**: Gen 1 (S0 and S1) and Gen 2 pricing tiers.\n\nGet locations of an individual government-issued tropical storms. Information about the tropical storms includes, location coordinates, geometry, basin ID, date, wind details and wind radii.",
```

Comment:
Change:
Get locations of an individual government-issued tropical storms.

To:
Get location of individual government-issued tropical storms.

Reply:
Updated!